release_notes/371-release-notes.adoc

```diff
@@ -70,9 +70,9 @@ Individual widget customization settings are short lived and are reset to the sy
 
 The Dashboard provides the following actionable widgets:
 
-* *Policy violations by severity*: This widget shows the distribution of violations across severity levels for the Dashboard filtered by scope. Clicking a *severity level* in the chart takes you to the *Violations* page, filtered for that severity and scope. It also lists the three most recent violations of a *Critical* level policy within the scope you defined in the Dashboard filter. Clicking a specific violation takes you directly to the *Violations* detail page for that violation.
+* *Policy violations by severity*: This widget shows the distribution of violations across severity levels for the Dashboard-filtered scope. Clicking a *severity level* in the chart takes you to the *Violations* page, filtered for that severity and scope. It also lists the three most recent violations of a *Critical* level policy within the scope you defined in the Dashboard filter. Clicking a specific violation takes you directly to the *Violations* detail page for that violation.
 
-* *Images at most risk*: This widget lists the top six vulnerable images within the Dashboard filtered by scope, sorted by their computed risk priority, along with the number of critical and important CVEs they contain. Click an image name to go directly to the *Image Findings* page under *Vulnerability Management*. Use the *Options* menu to focus on fixable CVEs, or further focus on active images. *Note*: When any option has been selected for the Dashboard filter, the data displayed is already filtered to active images, or images that are being used by deployments within the filtered scope. 
+* *Images at most risk*: This widget lists the top six vulnerable images within the Dashboard-filtered scope, sorted by their computed risk priority, along with the number of critical and important CVEs they contain. Click an image name to go directly to the *Image Findings* page under *Vulnerability Management*. Use the *Options* menu to focus on fixable CVEs, or further focus on active images. *Note*: When clusters or namespaces have been selected in the Dashboard filter, the data displayed is already filtered to active images, or images that are being used by deployments within the filtered scope. 
```

Comment:
I suggest that you either remove `*Note*:` or format the note like a note: 

```suggestion
* *Images at most risk*: This widget lists the top six vulnerable images within the Dashboard-filtered scope, sorted by their computed risk priority, along with the number of critical and important CVEs they contain. Click an image name to go directly to the *Image Findings* page under *Vulnerability Management*. Use the *Options* menu to focus on fixable CVEs or further focus on active images.
+
[NOTE]
====
When clusters or namespaces are selected in the Dashboard filter, the data displayed is already filtered to active images or images that are being used by deployments within the filtered scope. 
====
```

Reply:
TIL that you can keep the Note format under a bullet - thank you! Fixing now.